checklist_plus/text_generation/llm.py

```
@@ -0,0 +1,688 @@
+
+        self.tokenizer = self._create_dummy_tokenizer()
+
+    def _setup_default_llm(self, api_key: str | None, model_name: str) -> LLM:
+        """Setup default LLM client using OpenAI."""
+        if api_key is None:
+            api_key = os.getenv('OPENAI_API_KEY')
+            if not api_key:
+                raise ValueError(
+                    "OpenAI API key not provided. Set OPENAI_API_KEY environment variable "
+                    "or pass openai_api_key parameter."
+                )
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)